CefSharp/WebBrowserExtensions.cs

```diff
@@ -202,24 +203,21 @@ public static void ExecuteScriptAsync(this IWebBrowser browser, string methodNam
                 for (int i = 0; i < args.Length; i++)
                 {
                     var obj = args[i];
-                    if(obj == null)
+                    if (obj == null)
                     {
                         stringBuilder.Append("null");
                     }
+                    else if (numberTypes.Contains(obj.GetType()))
+                    {
+                        stringBuilder.Append(string.Format(CultureInfo.InvariantCulture, "{0}", args[i]));
```

Comment:
What about `Convert.ToString`? `string.format` whilst I'm sure works seems like a strange choice here (also not particularly efficient when run in a tight loop).